app/robots/iCubGenova04/XsensRetargeting.ini

```diff
@@ -3,20 +3,13 @@ samplingTime            0.01
 robot                   icub
 
 remote_control_boards   ("head", "torso", "left_arm", "right_arm")
-# Notice the order of the joint list is not wrong.
+# ROBOT JOINT LIST (Notice the order of the joint list is not wrong)
 # Indeed they are written according to the joint order of the walking-coordinator
 joints_list             ("neck_pitch", "neck_roll", "neck_yaw",
                          "torso_pitch", "torso_roll", "torso_yaw",
                          "l_shoulder_pitch", "l_shoulder_roll", "l_shoulder_yaw", "l_elbow", "l_wrist_prosup", "l_wrist_pitch", "l_wrist_yaw",
                          "r_shoulder_pitch", "r_shoulder_roll", "r_shoulder_yaw", "r_elbow",  "r_wrist_prosup", "r_wrist_pitch", "r_wrist_yaw")
 
-# Notice the order of the human joint list is not wrong.
```

Comment:
Is this removed because now the name of the joints are streamed from the wearable device`?

Reply:
yes.